src/CalcManager/CalculatorHistory.cpp

```diff
@@ -35,16 +35,14 @@ unsigned int CalculatorHistory::AddToHistory(_In_ shared_ptr<CalculatorVector <p
 
 unsigned int CalculatorHistory::AddItem(_In_ shared_ptr<HISTORYITEM> const &spHistoryItem)
 {
-    int lastIndex;
-
     if (m_historyItems.size() >= m_maxHistorySize)
     {
         m_historyItems.erase(m_historyItems.begin());
     }
 
     m_historyItems.push_back(spHistoryItem);
-    lastIndex = static_cast<unsigned>(m_historyItems.size() - 1);
-    return lastIndex;
+unsigned int lastIndex = static_cast<unsigned>(m_historyItems.size() - 1);
```

Comment:
ok...still learning how that suggested change feature works.  Looks like the indentation got lost when you accepted the change.  Let's try this again:
```suggestion
    unsigned int lastIndex = static_cast<unsigned>(m_historyItems.size() - 1);
    return lastIndex;
```

Reply:
Sadly, a limitation of GitHub's UI is ability to only suggest a change to single line. You can suggest inserting multiple lines, but it will always remove just one. If your suggestion were to be merged, there would be two return statements. An alternative would be doing two suggestions, but they will create two commits and thus two CI runs